src/main/java/LaunchClass.java

```diff
@@ -0,0 +1,35 @@
+import manager.TravelManager;
```

Comment:
why do you have this file in default package? why don't you have parent package with groupId, e.g. ua.lviv.iot, and then models -> ua.lviv.iot.models